src/custom/state/orders/updaters/GpOrdersUpdater.ts

```diff
@@ -86,14 +88,16 @@ function _transformGpOrderToStoreOrder(
     creationTime,
     summary: '',
     status,
-    receiver,
+    receiver: receiver || '', // TODO: check it
```

Comment:
@shoom3301 don't forget about this pls 🙏

Reply:
Thank you!
Remove TODO, it works fine